src/normalization.md

```diff
@@ -166,7 +166,9 @@ In this example:
 
 When interfacing with the type system it will often be the case that it's necessary to request a type be normalized. There are a number of different entry points to the underlying normalization logic and each entry point should only be used in specific parts of the compiler.
 
-An additional complication is that the compiler is currently undergoing a transition from the old trait solver to the new trait solver. As part of this transition our approach to normalization in the compiler has changed somewhat significantly, resulting in some normalization entry points being "old solver only" slated for removal in the long-term once the new solver has stabilized.
+An additional complication is that the compiler is currently (as of May 2025) undergoing a transition from the old trait solver to the new trait solver.
```

Comment:
you can choose to hide the date using notation in [this section], and it will appear in a monthly issue showing things that need refreshing ([example](https://github.com/rust-lang/rustc-dev-guide/issues/2370))

Reply:
Done.